spec.html

```diff
@@ -6460,7 +6460,7 @@ <h1>
         <dd>It is used to call the [[Call]] internal method of a function object. _F_ is the function object, _V_ is an ECMAScript language value that is the *this* value of the [[Call]], and _argumentsList_ is the value passed to the corresponding argument of the internal method. If _argumentsList_ is not present, a new empty List is used as its value.</dd>
       </dl>
       <emu-alg>
-        1. If _argumentsList_ is not present, set _argumentsList_ to a new empty List.
+        1. If _argumentsList_ is not present, set _argumentsList_ to &laquo; &raquo;.
```

Comment:
I can certainly do that - since i'd already converted everything to the syntax, I made the changes from there so all "Return &laquo; &raquo;" became "Return a new empty List". Is there a strong reason to prefer "status quo" here (where it's not in an argument position or returned), rather than just defaulting to "always use syntax in steps"?

Reply:
I generally prefer leaving things alone unless there's some reason to change them.

Reply:
My reason was that it increases consistency in the absence of a rule for this case. If you all still prefer I revert them, I can certainly do so. @michaelficarra @syg, thoughts?

Reply:
I was completely neutral on these cases. I do want to keep the non-empty ones though, those are nice.